### PR TITLE
[DUOS-2264][risk=no]Expand collapsible table width on all DAR tables 

### PR DESCRIPTION
**Pull request by kaitlynoflaherty**
## Adresses 
https://broadworkbench.atlassian.net/jira/software/c/projects/DUOS/boards/123?modal=detail&selectedIssue=DUOS-2264

- Displays the collapsible DAR ‘Data Use Group’ table with the same width as its parent row on all Console DAR Tables
- Decreases the spacing between the column titles and the first row in both the primary and secondary tables by 50%

**I'm  a little confused about the following instruction, but I think it looks good. Let me know if you think it needs updates: 
- The increased row width should be ~70% allocated to increasing the width of the Datasets column, and ~30% allocated to the Data Use Group column. 

image

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/libs/theme.js [hunk 1]
```diff
@@ -187,11 +187,11 @@ export const Styles = {
       fontFamily: 'Montserrat',
     },
     CARDCONTAINER: {
-      marginTop: '2rem',
+      margin: '2rem -12.5%',
```
Comment (rushtong) [inline src/libs/theme.js]: Can we tighten up the border between the top of the opened table with the row above it? Locally, I was able to get those to line up with `margin: '-.75rem -12.5%',`. However, `rem` is somewhat flexible and could change in different browsers. Might be nice to modify the border radius on the top of the table and bottom of the summary row to make them look even more tightly coupled.

## Original
Screenshot 2023-02-09 at 7 17 04 AM

## Updated
Screenshot 2023-02-09 at 7 22 17 AM

**Review by rushtong — APPROVED** [review]
See optional suggestions inline, implement as you see fit, but as is, I think this is a nice improvement 👍🏽

File: src/libs/theme.js [hunk 2]
```diff
@@ -187,11 +187,10 @@ export const Styles = {
       fontFamily: 'Montserrat',
     },
     CARDCONTAINER: {
-      marginTop: '2rem',
+      margin: '-.75rem -12.5%',
```
Comment (fboulnois) [inline src/libs/theme.js]:
```suggestion
      margin: '-.75rem -12.5% 2rem',
```

I think it's still worth visually separating the opened table from the row **below** it. So I'd recommend something like this.

**Review by fboulnois — CHANGES_REQUESTED** [review]
One recommendation:

**Review by fboulnois — APPROVED** [review]

**Pull request merged**